proxy/http3/Http3Transaction.cc

```diff
@@ -369,6 +369,9 @@ Http3Transaction::state_stream_open(int event, void *edata)
     int64_t len = this->_process_read_vio();
     // if no progress, don't need to signal
     if (len > 0) {
+      if (event == VC_EVENT_READ_COMPLETE) {
```

Comment:
IIRC, STREAM Frames of transport could be "pure FIN" - i.e. not payload.
In that case Http3Transaction receive `VC_EVENT_READ_COMPLETE` from QUICStream and no read?
If so, it looks like this block should be called even if `len` is zero. ( the comment in the line 370 might be wrong )

Reply:
Updated.